_docs/tasks/basic-access-control.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Enabling Simple Access Control
+overview: This task shows how to use Istio to control access to a service.
+          
+order: 41
+
+layout: docs
+type: markdown
+---
+
+This task shows how to use Istio to control access to a service.
+
+## Before you begin
+
+* Setup Istio by following the instructions in the
+  [Installation guide](/docs/tasks/installing-istio.html).
+
+* Deploy the [bookinfo](/docs/samples/bookinfo.html) sample application.
+
+* Initialize the application version routing by either first doing the
+  [request routing](/docs/tasks/request-routing.html) task or by running following
+  commands:
+  
+  ```bash
+  istioctl create -f route-rule-all-v1.yaml
+  istioctl replace -f route-rule-reviews-v2-v3.yaml
+  ```
+* Ensure that you can use [istioctl mixer](/docs/reference/commands/istioctl/istioctl_mixer.html#synopsis) by setting up port forwarding if needed.
+
+## Access control using `denials` 
+
+Using Istio you can control access to a service based on any attributes that are available within Mixer.
+This simple form of access control is based on conditionally denying requests using Mixer selectors.
+
+Consider the [bookinfo](/docs/samples/bookinfo.html) sample application where `ratings` service is accessed by multiple versions
+of the `reviews` service. We would like to cut off access to version `v3` of this service. 
+```bash
+istioctl mixer rule create global ratings.default.svc.cluster.local -f deny-reviews.yml
+```
+where deny-reviews.yml is 
+ 
+```yaml
+rules:
+- selector: source.labels["app"]=="reviews" && source.labels["version"] == "v3"  
+  aspects:
+  - kind: denials
+```
+This rule uses the `denials` aspect to deny requests coming from version `v3` of the reviews service.
```

Comment:
do we want some "Understanding" subsection (or full section below) that explains the denials aspect?

Also, can we link somewhere to a reference guide on denials aspect config?